paasta_tools/check_chronos_jobs.py

```diff
@@ -90,6 +90,23 @@ def sensu_event_for_last_run_state(state):
         return pysensu_yelp.Status.OK
 
 
+def filter_to_most_recent_jobs(jobs):
+    merged = {}
+    for job in jobs:
+        service, instance = chronos_tools.decompose_job_id(job['name'])
+        if instance not in merged:
+            merged[instance] = job
```

Comment:
It sounds like this function makes the assumption that it will only receive jobs for a particular service, right?

That sounds a little dangerous and it wasn't totally obvious that it worked like that, can you at least add a docstring or change the function name?